edge-modules/azure-monitor/src/MetricFilter.cs

```diff
@@ -65,14 +71,21 @@ public class MetricFilter
         /// Returns true if the passed metric matches a selector in this filter.
         /// Returns false if no metrics are in this filter.
         /// </summary>
-        public bool Matches(Metric testMetric) {
-            foreach (SingleMetricMatcher matcher in this.matchers) {
+        public bool Matches(Metric testMetric)
+        {
+            foreach (SingleMetricMatcher matcher in this.matchers)
+            {
                 if (matcher.Matches(testMetric))
                     return true;
             }
             return false;
         }
 
+        public override string ToString()
+        {
+            return this.metricList;
+        }
```

Comment:
Looks like this module was never integrated with our stylecop. Now whenever I make changes it gets auto-formatted to the correct format.

The autoformatting is making it hard to tell what I added, but I added this ToString() method and this instance variable. This is necessary so that when we print the configuration the module is starting with, it shows in some meaningful format rather than just the class name.